### DISCUSSION TITLE
Error Method Filament\Forms\Components\Toggle::dehydratedWhenHidden does not exist.

### DISCUSSION THREAD

**Discussion by atenadadkhah** [awcodes/filament-tiptap-editor#381] [Q&A] [answered]
When I click on some buttons in the editor like source code or media it throws this error `Method Filament\Forms\Components\Toggle::dehydratedWhenHidden does not exist.` 

Here is my code:
```PHP
    public static function form(Form $form): Form
    {
        return $form
            ->schema([
                Section::make()->schema([
                    TextInput::make('title')
                        ->label('title'),
                    TiptapEditor::make('description')
                        ->label('description')
                        ->required(),
                ]),
            ]);
    }
```

What is the problem??

**Comment by awcodes** [comment]
Try updating filament.

**Reply by atenadadkhah** [reply]
I'm using the last version

**Reply by awcodes** [reply]
Should be fine then. Also make sure you are on latest. 'php artisan about' what version does it show for filament?

**Reply by atenadadkhah** [reply]
It's 3.2.0

**Reply by awcodes** [reply, marked as answer]
You should definitely update then. Latest release is 3.2.76.

**Reply by atenadadkhah** [reply]
It worked, Thanks.
